css-positioning/relative.css

```diff
@@ -0,0 +1,50 @@
+h1 {
+  font-size: 4rem;
+}
+
+p {
+  font-size: 1.8em;
+  font-weight: 500;
+}
+
+.pokemon {
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 270px;
+
+  /* your code here */
+}
+
+#bulbasaur {
+  /* your code here */
+  position: relative;
```

Comment:
This won't have to be repeated for each Pokémon tag if it's move to line #18 (`img` selector).

Reply:
I have updated the css file. Thank you for the note!